packages/gerber-parser/lib/_parse-drill.js

```diff
@@ -63,6 +65,10 @@ var parseCommentForFormatHints = function(parser, block, line) {
     var altiumMatch = block.match(RE_ALTIUM_HINT)
 
     result.places = [Number(altiumMatch[1]), Number(altiumMatch[2])]
+  } else if (RE_ALTIUM_PTH.test(block)) {
+    parser._push(commands.set('plating', 'PTH', line))
```

Comment:
A couple things:

- Could this setting be called `drillPlating`, since it doesn't have any meaning in the context of a Gerber file
- Can you think of any reason why the value shouldn't be a boolean (i.e. plated > true, non-plated > false) rather than an arbitrary string constant?

Reply:
`drillPlating` is a little strange. Sounds like we are plating the drill and not the hole :). 
`holePlating` could be a suitable alternative.

We actually started with `holePlating` but then switched to `plating` to avoid the whole snake case vs camel case conversation in the output JSON.

Boolean is a valid consideration. We went with the string constant since it is extensible without changing the type of the data. But realistically cannot see any specific case at this stage that would require extension.

`PTH` and, `NPTH` are pretty industry standard terminology, including for Chinese CAM and PCB design guys.

Reply:
`holePlating` is fine by me. Convention for this project is camel-case (see examples in the API docs)

I don't have strong feelings on string enum vs boolean, it was just something that popped into my head. If you decide to go with the string enum, I'd ask that you lowercase it. I made the mistake of not being consistent about that early on with this project, and now some existing values are lowercase (e.g. `mode: 'ccw'`) and some are not (e.g. `nota: 'A'`), which bothers me 🙃

Reply:
Cool, will add the unit tests in the next day or so then update this PR.